quantum/quantum_keycodes.h

```diff
@@ -524,6 +524,40 @@ enum quantum_keycodes {
     // Additional magic key
     MAGIC_TOGGLE_GUI,
 
+    // Dedicated macro keys for Configurator, XAP, and VIA
```

Comment:
@fauxpark does anything special need to happen here, w.r.t. your keycode work?

If the intent is to allow for VIA support, then [static asserts](https://github.com/qmk/qmk_firmware/blob/master/quantum/via_ensure_keycode.h) should reflect the new keycodes, too.

```suggestion
    // Dedicated macro keys for Configurator, and VIA
```

Reply:
If I need to add static asserts I'll have to move these above sequencer. I guess that's probably ok?

Reply:
After digging into this I'm not going to assert these keycodes. VIA has its own set of macro keycodes which are well into `SAFE_RANGE`, and resolving that is a larger challenge than I can take on right now.

Reply:
```suggestion
    // Dedicated macro keys for Configurator and VIA
```